[2024-09-11T06:27:27.419+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-11T06:27:27.449+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: parse_fixed_width_to_csv.parse_fixed_width manual__2024-09-11T06:22:07.334823+00:00 [queued]>
[2024-09-11T06:27:27.471+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: parse_fixed_width_to_csv.parse_fixed_width manual__2024-09-11T06:22:07.334823+00:00 [queued]>
[2024-09-11T06:27:27.472+0000] {taskinstance.py:2865} INFO - Starting attempt 2 of 2
[2024-09-11T06:27:27.526+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): parse_fixed_width> on 2024-09-11 06:22:07.334823+00:00
[2024-09-11T06:27:27.537+0000] {standard_task_runner.py:72} INFO - Started process 327 to run task
[2024-09-11T06:27:27.549+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'parse_fixed_width_to_csv', 'parse_fixed_width', 'manual__2024-09-11T06:22:07.334823+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/parse_fixed_width_dag.py', '--cfg-path', '/tmp/tmp_fvy5nku']
[2024-09-11T06:27:27.555+0000] {standard_task_runner.py:105} INFO - Job 11: Subtask parse_fixed_width
[2024-09-11T06:27:27.713+0000] {task_command.py:467} INFO - Running <TaskInstance: parse_fixed_width_to_csv.parse_fixed_width manual__2024-09-11T06:22:07.334823+00:00 [running]> on host 80f7b0fb747e
[2024-09-11T06:27:27.923+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='parse_fixed_width_to_csv' AIRFLOW_CTX_TASK_ID='parse_fixed_width' AIRFLOW_CTX_EXECUTION_DATE='2024-09-11T06:22:07.334823+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-11T06:22:07.334823+00:00'
[2024-09-11T06:27:27.924+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-11T06:27:28.004+0000] {taskinstance.py:3310} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/parse_fixed_width_dag.py", line 19, in run_parsing_script
    raise Exception(f"Script failed with error: {result.stderr}")
Exception: Script failed with error: python3: can't open file '/opt/airflow/dags/fixed_width_parser.py': [Errno 2] No such file or directory

[2024-09-11T06:27:28.031+0000] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=parse_fixed_width_to_csv, task_id=parse_fixed_width, run_id=manual__2024-09-11T06:22:07.334823+00:00, execution_date=20240911T062207, start_date=20240911T062727, end_date=20240911T062728
[2024-09-11T06:27:28.091+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-11T06:27:28.092+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 11 for task parse_fixed_width (Script failed with error: python3: can't open file '/opt/airflow/dags/fixed_width_parser.py': [Errno 2] No such file or directory
; 327)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 115, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3004, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3158, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3182, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/parse_fixed_width_dag.py", line 19, in run_parsing_script
    raise Exception(f"Script failed with error: {result.stderr}")
Exception: Script failed with error: python3: can't open file '/opt/airflow/dags/fixed_width_parser.py': [Errno 2] No such file or directory

[2024-09-11T06:27:28.127+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-09-11T06:27:28.195+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-11T06:27:28.202+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
